src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -49,13 +56,21 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
 
   const queryVariables = getQueryVariables(search, resourceId);
 
-  const { sortBy, sortDir, limit, page } = queryVariables;
+  let { sorts } = queryVariables;
+  const { limit, page } = queryVariables;
 
-  const columns = useSetColumnDefaultSortOrder<TaskResult>(
-    columnsTemplate,
-    sortBy,
-    sortDir
-  );
+  if (sorts.length === 0) {
+    sorts = [
```

Comment:
> This should be be a const.

sorts represents a value that exists in the url. by over writing it here without also updating it within the url we are invalidating the truthiness to this value. It no longer represents the state of the url. If you want to apply default sort parameters you should instead update the query params within the url.